packages/web/src/components/shared/ListenSvg.js

```diff
@@ -1,520 +1,522 @@
+				}
+				#el_DeebuCsPTGA {
+					-webkit-transform: translate(37.846924px, 0px);
+					transform: translate(37.846924px, 0px);
+					-webkit-animation-fill-mode: backwards;
+					animation-fill-mode: backwards;
+					opacity: 0;
+					-webkit-animation-name: kf_el_DeebuCsPTGA_an_aYTRBE7Na;
+					animation-name: kf_el_DeebuCsPTGA_an_aYTRBE7Na;
+					-webkit-animation-timing-function: cubic-bezier(0.42, 0, 1, 1);
+					animation-timing-function: cubic-bezier(0.42, 0, 1, 1);
+				}
+				#el__ZcqlS20zcw {
+					fill: #2196f3;
+				}
+				#el_8DnEQnD7VWV {
+					fill: #d8d8d8;
+				}
+				#el_ZOjjrPTvyrv {
+					-webkit-transform: translate(37.846924px, 0px);
+					transform: translate(37.846924px, 0px);
+					-webkit-animation-fill-mode: backwards;
+					animation-fill-mode: backwards;
+					opacity: 0;
+					-webkit-animation-name: kf_el_ZOjjrPTvyrv_an_l_BjBNzXw;
+					animation-name: kf_el_ZOjjrPTvyrv_an_l_BjBNzXw;
+					-webkit-animation-timing-function: cubic-bezier(0.42, 0, 1, 1);
+					animation-timing-function: cubic-bezier(0.42, 0, 1, 1);
+				}
+				#el_FYYKCI_u24e {
+					fill: #2196f3;
+				}
+				#el_XZty4MnTp5Y {
+					fill: #d8d8d8;
+				}
+				#el_2FATegVmf0K {
+					-webkit-transform: translate(37.846924px, 0px);
+					transform: translate(37.846924px, 0px);
+					-webkit-animation-fill-mode: backwards;
+					animation-fill-mode: backwards;
+					opacity: 0;
+					-webkit-animation-name: kf_el_2FATegVmf0K_an_wLg4ofuFx;
+					animation-name: kf_el_2FATegVmf0K_an_wLg4ofuFx;
+					-webkit-animation-timing-function: cubic-bezier(0.42, 0, 1, 1);
+					animation-timing-function: cubic-bezier(0.42, 0, 1, 1);
+				}
+				#el_RMT1KUfbdF8 {
+					fill: #2196f3;
+				}
+				#el_RgLcovvFiO1 {
+					fill: #d8d8d8;
+				}
+			`}
+		/>
+		<svg
+			viewBox="0 0 480 480"
+			xmlns="http://www.w3.org/2000/svg"
+			xmlnsXlink="http://www.w3.org/1999/xlink"
+			id="el_hiibMG0x-"
+			width={28}
+			height={29}
+			{...props}
+		>
+			<defs>
 				<path
-					d="M142.731204,111 C137.280427,111 132.719573,114.852 131.82965,120.095 C127.268796,145.24 104.464526,164.5 76.9881611,164.5 C49.5117965,164.5 26.7075263,145.24 22.1466723,120.095 C21.2567496,114.852 16.6958955,111 11.2451187,111 C4.45945784,111 -0.880078594,116.778 0.121084488,123.198 C5.57186127,155.298 32.2695435,180.443 65.8641269,185.044 L65.8641269,207.3 C65.8641269,213.185 70.8699423,218 76.9881611,218 C83.10638,218 88.1121954,213.185 88.1121954,207.3 L88.1121954,185.044 C121.706779,180.443 148.404461,155.298 153.855238,123.198 C154.967641,116.778 149.516864,111 142.731204,111 Z"
-					id="el_uzZNtK32Zi"
-					fillRule="nonzero"
-					style={{ fill: '#2196F3' }}
+					d="M38.779092,147.789474 C60.0824253,147.789474 77.279092,131.286316 77.279092,110.842105 L77.279092,36.9473684 C77.279092,16.5031579 60.0824253,0 38.779092,0 C17.4757586,0 0.279091964,16.5031579 0.279091964,36.9473684 L0.279091964,110.842105 C0.279091964,131.286316 17.4757586,147.789474 38.779092,147.789474 Z"
+					id="path-1"
```

Comment:
Why there are changes in the path definition attribute?

Reply:
@jyash97 check properly. It's the diff from the upper part